src/Pact/Runtime/Utils.hs

```diff
@@ -188,16 +192,21 @@ lookupModule i mn = do
             MDModule m -> GPostRead (ReadModule (_mName m) (_mCode m))
             MDInterface int -> GPostRead (ReadInterface (_interfaceName int) (_interfaceCode int))
           natives <- view $ eeRefStore . rsNatives
-          let natLookup (NativeDefName n) = case HM.lookup (Name (BareName n def)) natives of
+          let natLookup (NativeDefName n) = case HM.lookup n natives of
                 Just (Direct t) -> Just t
                 _ -> Nothing
           case traverse (traverse (fromPersistDirect natLookup)) mdStored of
             Right md -> do
               evalRefs . rsLoadedModules %= HM.insert mn (md,False)
+              loadModuleDependencies md
```

Comment:
To be clear about this change: loading modules for use and execution requires transitive dependencies to be present and loaded.